firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/UpdateProgress.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.appdistribution;
+
+import androidx.annotation.NonNull;
+
+/** Interface to get download progress for APKs and the status of the update. Used in updateApp. */
+public interface UpdateProgress {
```

Comment:
same question here: why an interface rather than a POJO?

Reply:
Updated to a POJO